common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
+        if self.isEnabledFor(AUDIT):
+            self._log(AUDIT, message, args, **kws)
+
+    logging.Logger.audit = audit
+
+
+def load_global_log_config():
+    """
+    A general method to load the overall config of the system, specifically it modifies the root handler to output
+    to sysout and sets the default log levels and format. This is expected to be called once at the start of a
+    system.
+    """
+    _add_audit_level_log()
+    logger = logging.getLogger()
+    logger.setLevel(logging.NOTSET)
+    handler = logging.StreamHandler(sys.stdout)
+    logging.Formatter.converter = time.gmtime
+    formatter = logging.Formatter('[%(asctime)s.%(msecs)03d] '
+                                  '%(message)s pid=%(process)d LogLevel=%(levelname)s ProcessKey=%(processKey)s',
+                                  '%Y-%m-%d %H:%M:%S')
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+
+
+def _format_values_in_map(dict_values: dict) -> dict:
+    """
+    Replaces the values in the map with key=value so that the key in a string can be replaced with the correct
+    log format, also surrounds the value with quotes if it contains spaces
+    """
+    new_map = {}
+    for key, value in dict_values.items():
+        if ' ' in value:
+            value = f"\"{value}\""
+
+        new_map[key] = f"{key.replace(' ', '')}={value}"
+    return new_map
+
+
+def _formatted_string(message: str, dict_values: dict) -> str:
+    """
+    Populates the string with the correctly formatted dictionary values
+    """
+    formatted_values = _format_values_in_map(dict_values)
+    return message.format(**formatted_values)
+
+
+class IntegrationAdaptersLogger:
```

Comment:
```suggestion
class IntegrationAdaptorsLogger:
```
I know this is being pedantic, but I feel like might as well be consistent in our repo (and also to avoid any confusion). And yes, I did Google [the difference between the 2 words](https://grammarist.com/spelling/adapter-adaptor/): apparently both of them are fine.